### PR TITLE
Release: 1.2.1

### LINKED ISSUE #47: Fix gemspec / deps to remove warnings
There are a bunch of simple warnings that we should fix:

```
Russells-MacBook:rainforest-cli russ$ gem build rainforest-cli.gemspec 
WARNING:  description and summary are identical
WARNING:  open-ended dependency on httparty (>= 0) is not recommended
  if httparty is semantically versioned, use:
    add_runtime_dependency 'httparty', '~> 0'
WARNING:  open-ended dependency on parallel (>= 0) is not recommended
  if parallel is semantically versioned, use:
    add_runtime_dependency 'parallel', '~> 0'
WARNING:  open-ended dependency on ruby-progressbar (>= 0) is not recommended
  if ruby-progressbar is semantically versioned, use:
    add_runtime_dependency 'ruby-progressbar', '~> 0'
WARNING:  open-ended dependency on rainforest (>= 0) is not recommended
  if rainforest is semantically versioned, use:
    add_runtime_dependency 'rainforest', '~> 0'
WARNING:  open-ended dependency on rake (>= 0, development) is not recommended
  if rake is semantically versioned, use:
    add_development_dependency 'rake', '~> 0'
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
  Successfully built RubyGem
  Name: rainforest-cli
  Version: 1.2.0
  File: rainforest-cli-1.2.0.gem
Russells-MacBook:rainforest-cli russ$ gem push rainforest-cli-1.2.0.gem 
Pushing gem to https://rubygems.org...
Successfully registered gem: rainforest-cli (1.2.0)
Russells-MacBook:rainforest-cli russ$
```

### PR DESCRIPTION
**Pull request by epaulet**
fixes #47 and prep for release of bug fix

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,2 +1,6 @@
 inherit_gem:
   rf-stylez: ruby/rubocop.yml
+
+AllCops:
+  Exclude:
+    - rainforest-cli.gemspec
```
Comment (shosti) [inline .rubocop.yml]: Why not just fix the gemspec?

File: rainforest-cli.gemspec [hunk 2]
```diff
@@ -18,10 +18,10 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "httparty"
-  spec.add_dependency "parallel"
-  spec.add_dependency "ruby-progressbar"
-  spec.add_dependency "rainforest"
+  spec.add_dependency "httparty", "0.13.7"
```
Comment (shosti) [inline rainforest-cli.gemspec]: I think all of these should be `~>`. Otherwise they might mess up dependency fetching for projects that require it.

**Comment by shosti** [comment]
@epaulet can you actually fix the dependency thing and re-release before pushing?

**Comment by epaulet** [comment]
Sure.

**Pull request merged**